academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -15,12 +16,12 @@ Creates the MAKES index hosting resources. E.g. MAKES web host image
 
 ```cmd
 kesm CreateHostResources --HostResourceName
-            --MakesPackage
-            --[Region]
-            --[MakesWebHost]
+                         --MakesPackage
+                         [--Region]
+                         [--MakesWebHost]
 ```
 
-### Required parameters
+### CreateHostResources required parameters
 
 `--HostResourceName`
```

Comment:
Something might be screwed up with the diff tool, but shouldn't --HostResourceName and --Region have descriptions?

Reply:
Line 34 should have it

Reply:
Yep, for some reason it's not showing in the diff tool...

---
In reply to: [465200315](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/172#discussion_r465200315) [](ancestors = 465200315)